docs/modules/guides/pages/form-view/creating-a-fileupload-form.adoc

```diff
@@ -70,65 +58,60 @@ class DummyEntityConfiguration implements EntityConfigurer {
     public void configure(EntitiesConfigurationBuilder entities) {
         entities.withType(DummyEntity.class)
             // Attach our custom  formView
-            .formView("upload", fvb -> fvb.viewProcessor(fileUploadViewProcessor)
+            .formView("upload", fvb -> fvb.viewProcessor(fileUploadViewProcessor) # <1>
 
                 // Create a new property files that will represent our file upload element
-                .properties(props -> props.property("files")
+                .properties(props -> props.property("files") # <2>
 
-                    // Configure a label for the file input field
+                    // Configure a label for the file input field # <3>
                     .displayName("Csv file")
 
                     // Set the right propertyType
-                    .propertyType(MultipartFile[].class)
+                    .propertyType(MultipartFile[].class) # 
 
                     /** Set the right viewElementBuilder so entityModule knows how to render
                     * our file upload field. Note that we didn't use .viewElementType that will autodetect
                     * the viewElementBuilder to use.
                     */
-                    .viewElementBuilder(
+                    .viewElementBuilder( # <4>
                         ViewElementMode.CONTROL, BootstrapUiBuilders.file()
                             .controlName("extensions[upload].files")
                     )
 
                     // We need our new property to be writable
-                    .writable(true)
+                    .writable(true) # <5>
                 )
                 // We want to be sure our property is shown on our formView
                 .showProperties("files")
 
                 // Remove the default viewProcessor SaveEntityViewProcessor so that only our form get's saved
                 // and not the attached entity
-                .removeViewProcessor(SaveEntityViewProcessor.class.getName())
+                .removeViewProcessor(SaveEntityViewProcessor.class.getName()) # <5>
             );
     }
  }
 ----
 
-The above code registers a new formView with the name upload and add's our created but not yet implemented viewHelper
+<1> The above code registers a new formView with the name upload and add's our created but not yet implemented viewHelper
```

Comment:
There is no reference needed to the code snippet as this is a callout from within the code snippet.
Clarity in naming, An `EntityViewProcessor` is added, not a viewHelper, preference is to `EntityViewProcessor` or view processor